paddle/fluid/eager/backward.cc

```diff
@@ -294,6 +307,117 @@ class GeneralGrad {
     return results;
   }
 
+  void SetNodeToAccumulationNode(GradNodeBase* node) {
```

Comment:
Potential stop nodes may be don't needed. The simple logic is 1. all input target node should become accumulation node 2. some input target is others' preceding nodes.

Reply:
Done, Potential stop nodes is removed